app/views/layouts/application.html.slim

```diff
@@ -39,6 +39,9 @@ html
         = yield
   footer
     .aside-footer
+      p
+        - unless Git::BRANCH.empty?
+          '#{Git::BRANCH}@#{Git::COMMIT}
```

Comment:
Maybe it would be better that the `Git` module provided a method named `version` or something, thus moving the logic to the ruby code. Plus, I'd also want to see the commit even if the branch is empty. This can happen for example by performing the following commands:

```
$ git remote add lonewulf https://github.com/lonewulf/Portus
$ git fetch lonewulf
$ git checkout lonewulf/669_git_webgui
```

In this case `BRANCH` is empty but `COMMIT` is not.